.gitignore

```diff
@@ -14,4 +14,5 @@ mirage/*/tls_server
 Makefile
 key_gen.ml
 
-random/
\ No newline at end of file
+random/
+vendor/
```

Comment:
You don't need this any more either...

Reply:
hmmm ... git seems to want to include things in this folder and it always lists it when doing `git status`. Of course you can do selective `git add ...` but mostly I prefer the convenience of `git add --all`. This prevents from mistakenly adding things from `vendor` dir  to this repo.